parachain/pallets/erc20-app/src/lib.rs

```diff
@@ -125,10 +127,12 @@ impl<T: Trait> Module<T> {
 }
 
 impl<T: Trait> Application for Module<T> {
-	fn handle(payload: &[u8], _verification_output: &VerificationOutput) -> DispatchResult {
-		let payload_decoded = InPayload::decode(payload)
+	fn handle(payload: &[u8]) -> DispatchResult {
```

Comment:
stupid question likely: it's not possible to call this handle() function directly from an extrinsic, bypassing the bridge - the only way to call it is via the bridge submit method, right? @vgeddes @Rizziepit

Reply:
Correct